library/src/scripts/styles/globalStyleVars.ts

```diff
@@ -10,6 +10,8 @@ export const globalVariables = (theme?: object) => {
     const colorPrimary = color("#0291db");
     const themeVars = componentThemeVariables(theme, "globalVariables");
 
+    const baseUnit = 6;
```

Comment:
This doesn't really apply anymore, we don't really use it to scale things and I find it's less easy to read than a real value. I was getting rid of it.